js/tests/loadtest.ts

```diff
@@ -120,6 +121,14 @@ const FEE_DIVISOR = 50;
 
 })();
 
+async function createRandomZKSyncWallet(ethWallet: ethers.Wallet, provider: Provider, ethProxy: ETHProxy) {
```

Comment:
why did you add this? this should be deterministic. please revert back to `fromEthSigner` where signers are derived from mnemonic.

Reply:
`fromEthSigner` func from zksync.js uses a static seed to generate a private key. And in order to be able to run the loadtest in several terminal tabs at the same time, I need different accounts in order to avoid a nonce collision. Therefore, I wrote a random seed generation in this function